proto/ostracon/abci/types.proto

```diff
@@ -357,6 +357,9 @@ message Validator {
   bytes address = 1;  // The first 20 bytes of SHA256(public key)
   // PubKey pub_key = 2 [(gogoproto.nullable)=false];
   int64 power = 3;  // The voting power
```

Comment:
Could you update the comment `The voting power` to `The staking power`? and also put the comment `The staking power` in the `message ValidatorUpdate.power`?

Reply:
fixed